components/genetics.csl

```diff
@@ -151,7 +151,7 @@
           </group>
         </else>
       </choose>
-      <text macro="access" prefix=". "/>
+      <text variable="url" display="block" prefix=" url: "/>
```

Comment:
I think we want to leave this as is, and modify the `access` macro. This will allow us to display the DOI if it exists.

```suggestion
      <text macro="access" prefix=". "/>
```

I think the following update to the access macro should do the trick (I used https://editor.citationstyles.org/visualEditor/ to work this out)
```<macro name="access">
    <choose>
      <if match="any" variable="DOI">
        <text variable="DOI" prefix="https://doi.org/"/>
      </if>
      <else-if match="any" variable="URL">
        <text variable="URL"/>
      </else-if>
    </choose>
  </macro>
```

Reply:
I may have put this in the wrong place? It's not working

Reply:
I'm not familiar with how this stuff works, I just made changes until it did.

Reply:
Would you mind committing this change where it's supposed to go? I'm a bit confused on where it should go.

Reply:
Done. (It replaced the current access macro)

Reply:
Thanks!

Reply:
It works!